Movie Challenge/Movie Challenge/Controllers/Search/SearchViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  SearchCollectionViewController.swift
+//  Movie Challenge
+//
+//  Created by igor gomes arantes on 03/09/18.
+//  Copyright © 2018 igor gomes arantes. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SearchViewController : UIViewController, UICollectionViewDelegate, UICollectionViewDataSource, UISearchBarDelegate{
```

Comment:
To organize better you have to implement the Delegates and DataSources inside extensions blocks.